terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -0,0 +1,29 @@
+environment = "live"
+aws_profile = "live-eu-west-2"
+kms_alias = "alias/live-eu-west-2/environment-services-kms"
```

Comment:
Please check with Ellis about `kms_alias`. If it is common for each team then it is fine otherwise we need to check what is our alias name.